heartbeat/beater/heartbeat.go

```diff
@@ -131,6 +131,11 @@ func New(b *beat.Beat, rawConfig *conf.C) (beat.Beater, error) {
 		}),
 		trace: trace,
 	}
+	runFromID := "<unknown location>"
```

Comment:
nit: we have `default` value as runFrom Id in state and `unknown` here, can we follow a common value.